### PR TITLE
docs: remove addTags method

### PR DESCRIPTION
**Pull request by jahtalab**
Part of #272 and #215 

* mention support fo boolean and number values for addLabels

### CODE REVIEW

File: docs/agent-api.asciidoc [hunk 1]
```diff
@@ -98,7 +92,8 @@ Arguments:
 
 * `name` - Any string. All periods (.), asterisks (*), and double quotation marks (") will be replaced by underscores (_), as those characters have special meaning in Elasticsearch
 
-* `value` - Any string. If a non-string data type is given, it's converted to a string before being sent to the APM Server.
+* `value` - Any string, boolean or number. If a data type other than the previously mentioned types
+ is given, it's converted to a string before being sent to the APM Server.
```
Comment (bmorelli25) [inline docs/agent-api.asciidoc]:
```suggestion
* `value` - Any string, boolean, or number. All other data types will be converted to a string
before being sent to the APM Server.
```

File: docs/span-api.asciidoc [hunk 2]
```diff
@@ -61,8 +54,8 @@ Arguments:
 
 * `name` - Any string. All periods (.), asterisks (*), and double quotation marks (") will be replaced by underscores (_), as those characters have special meaning in Elasticsearch
 
-* `value` - Any string. If a non-string data type is given, it's converted to a string before being sent to the APM Server.
-
+* `value` - Any string, boolean or number. If a data type other than the previously mentioned types
+ is given, it's converted to a string before being sent to the APM Server.
```
Comment (bmorelli25) [inline docs/span-api.asciidoc]:
```suggestion
* `value` - Any string, boolean, or number. All other data types will be converted to a string
before being sent to the APM Server.
```

File: docs/transaction-api.asciidoc [hunk 3]
```diff
@@ -100,7 +94,8 @@ Arguments:
 
 * `name` - Any string. All periods (.), asterisks (*), and double quotation marks (") will be replaced by underscores (_), as those characters have special meaning in Elasticsearch
 
-* `value` - Any string. If a non-string data type is given, it's converted to a string before being sent to the APM Server.
+* `value` - Any string, boolean or number. If a data type other than the previously mentioned types
+ is given, it's converted to a string before being sent to the APM Server.
```
Comment (bmorelli25) [inline docs/transaction-api.asciidoc]:
```suggestion
* `value` - Any string, boolean, or number. All other data types will be converted to a string
before being sent to the APM Server.
```

**Comment by vigneshshanmugam** [comment]
Test failure is unrelated.

**Pull request merged**